app/mailers/user_mailer.rb

```
@@ -1,15 +1,19 @@
 class UserMailer < ApplicationMailer
 	default from: 'er.moonlighter.scheduler.com'
 
-	def dummy_email(doctor, subject, text)
+	def custom_email(doctor, subject, text)
 		@user = doctor
 		@text = text
-		mail(to: @user.email, subject: subject)
+		if @user.email
```

I don't think this should fail silently. Everyone should have an email because they signed up for the site, so if this case doesn't happen it's a serious answer and we should alert it (even it it's just so that the admin knows their email wasn't sent)